GetJobber/atlantis
# chore(components): JOB-75752 DataList responsivenes

## Motivations



## Changes



### Added

- Ability to add `DataListLayouts` for various breakpoints
- Added ability to hide the header on specific breakpoints

### Changed

- 

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

 - Verify the layout changes based on the size prop for `DataList.Layout`s
 - Verify that the `header` can be hidden for specific `DataList.Layout`s

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)
